modules/cluster-logging-loki-deploy.adoc

```diff
@@ -101,12 +101,16 @@ oc apply -f logging-loki.yaml
   apiVersion: logging.openshift.io/v1
   kind: ClusterLogging
   metadata:
-    name: cr-lokistack
+    name: instance
     namespace: openshift-logging
   spec:
-     logStore:
+    managementState: Managed
+    logStore:
+       type: lokistack
        lokistack:
          name: logging-loki
+    collection: 
+        type: "vector"
```

Comment:
link 113. Please remove two prefix spaces.

Reply:
The other part /lgtm

Reply:
Done :) ty! look good?

Reply:
line 113, not 133